activities/cleanup_ingest.py

```diff
@@ -129,19 +133,58 @@ def run(self):
         finally:
             db_connection.close()
 
+    def delete_lambda_event_source_queues(self):
+        """
+        Remove SQS queues connected to ingest lambdas that are connected as
+        event sources.
+        """
+        if int(self.job['ingest_type']) != TILE_INGEST:
+            return
+
+        try:
+            config = BossConfig()
+            INGEST_LAMBDA = config["aws"]["tile_ingest_lambda"]
+            TILE_UPLOADED_LAMBDA = config["aws"]["tile_uploaded_lambda"]
+        except Exception as ex:
+            log.error(f'Failed to get lambda names from boss.config: {ex}')
+            return
+
+        self.remove_sqs_event_source_from_lambda(TileIndexQueue(self.nd_proj).arn, TILE_UPLOADED_LAMBDA)
```

Comment:
Will these only delete for a single ingest job.  In other words will this act correctly if we had two different ingest-jobs going at the same time?

Reply:
Yes, the cleanup runs for just one ingest job and deletes its resources. So no problem if multiple jobs enter cleanup simultaneously.